packages/core-data/src/queried-data/get-query-parts.js

```diff
@@ -80,6 +80,9 @@ export function getQueryParts( query ) {
 
 				// Two requests with different include values cannot have same results.
 				if ( key === 'include' ) {
+					if ( typeof value === 'number' ) {
+						value = value.toString();
+					}
```

Comment:
I'd prefer to have this in `getNormalizedCommaSeparable` which already handles types conversion.

Reply:
Actually, thinking about this more, `getNormalizedCommaSeparable` is also used in a few different places where such behavior could be unexpected. Let's move forward with this version then. Thank you for contributing!